### PR TITLE
:wrench: add BI information

### PR DESCRIPTION
**Pull request by yuwzho**

### CODE REVIEW

**Comment by yuwzho** [comment]
@xscript

File: index.js [hunk 1]
```diff
@@ -115,6 +116,10 @@ function initClient(connectionStringParam, credentialPath) {
   wpi.pinMode(config.LEDPin, wpi.OUTPUT);
   messageProcessor = new MessageProcessor(config);
 
+  bi.start()
+  bi.trackEvent('success');
```
Comment (xscript) [inline index.js]: what about errors and failures?

Comment (yuwzho) [inline index.js]: currently, we only note how many people already run this sample

Comment (xscript) [inline index.js]: Got it. Just want to remind you the possibility of more event types in the future.

**Pull request merged**